helm/charts/keto/README.md

```diff
@@ -8,88 +8,89 @@ Access Control Policies as a Server
 
 ## Maintainers
```

Comment:
Those files are auto-generated by helm-docs plugin from our values.yaml. The `make format` and the plugin will override each other all the time 😞